### ISSUE TITLE
AR IT:  Nightly report - Failed `test_execd_restart`

### ISSUE THREAD

**Issue by Rebits** [wazuh/wazuh-qa#2769]
|Related issue|
|---|
|https://github.com/wazuh/wazuh-qa/issues/2770| 
|https://github.com/wazuh/wazuh-qa/issues/3022| 

| **Date** |  **Commit** | Build | 
| --- | --- |--- |
| 2022-04-06 | 99be0c9 | https://ci.wazuh.info/job/test_nightly/
| 2022-06-28 | 4a0c134| https://ci.wazuh.info/job/test_nightly/1077//

###  `test_execd_restart`

- **Build**: https://ci.wazuh.info/job/Test_integration/24788/console

- **Cases**
```
FAILED test_active_response/test_execd/test_execd_restart.py::test_execd_restart[get_configuration0]
```
- **Description**

Windows agent does not restart after AR.
```
2022/04/07 08:14:25 wazuh-agent[3860] receiver-win.c:132 at receiver_thread(): DEBUG: Received message: '#!-execd {"version":1,"origin":{"name":"","module":"wazuh-analysisd"},"command":"restart-wazuh0","parameters":{"extra_args":[],"alert":{"rule":{"level":5,"description":"Test.","id":554}}}}'
2022/04/07 08:14:25 wazuh-agent[3860] agent_op.c:237 at os_read_agent_profile(): DEBUG: os_read_agent_profile() = [-]
2022/04/07 08:14:25 wazuh-agent[3860] config.c:407 at ReadConfig(): DEBUG: agent_config element does not have any attributes.
2022/04/07 08:14:25 wazuh-agent[3860] syscheck.c:118 at Start_win32_Syscheck(): INFO: (6678): No directory provided for syscheck to monitor.
2022/04/07 08:14:25 wazuh-agent[3860] syscheck.c:138 at Start_win32_Syscheck(): INFO: (6001): File integrity monitoring disabled.
2022/04/07 08:14:25 wazuh-agent[3860] agent_op.c:218 at os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().
2022/04/07 08:14:25 wazuh-agent[3860] agent_op.c:237 at os_read_agent_profile(): DEBUG: os_read_agent_profile() = [-]
2022/04/07 08:14:25 wazuh-agent[3860] config.c:407 at ReadConfig(): DEBUG: agent_config element does not have any attributes.
2022/04/07 08:14:25 rootcheck[3860] rootcheck.c:202 at rootcheck_init(): INFO: Rootcheck disabled.
2022/04/07 08:14:25 wazuh-agent[3860] syscheck.c:204 at Start_win32_Syscheck(): DEBUG: (6357): Maximum disk quota size limit configured to '1048576 KB'.
2022/04/07 08:14:25 wazuh-agent[3860] syscheck.c:259 at Start_win32_Syscheck(): INFO: Started (pid: 3860).
2022/04/07 08:14:25 wazuh-agent[3860] config.c:407 at ReadConfig(): DEBUG: agent_config element does not have any attributes.
2022/04/07 08:14:25 wazuh-agent[3860] receiver-win.c:132 at receiver_thread(): DEBUG: Received message: '#!-agent ack '
2022/04/07 08:14:25 wazuh-agent[3860] exec.c:65 at ReadExecConfig(): ERROR: (1313): Invalid active response config: 'shared/ar.conf'.
2022/04/07 08:14:25 wazuh-agent[3860] exec.c:105 at ReadExecConfig(): INFO: Active response command not present: 'active-response/bin/restart-wazuh'. Not using it on this system.
2022/04/07 08:14:25 sca[3860] wm_sca.c:153 at wm_sca_main(): INFO: Module disabled. Exiting.
```

**NOTE:** Reported by Nightly [2022-04-21](https://github.com/wazuh/wazuh-qa/issues/2811), [2022-04-29](https://github.com/wazuh/wazuh-qa/issues/2850), [2022-05-12](https://github.com/wazuh/wazuh-qa/issues/2879), [2022-05-18](https://github.com/wazuh/wazuh-qa/issues/2906)

**Comment by Rebits** [comment]
07/04/2022
Detected error in Windows agent https://github.com/wazuh/wazuh/issues/13077#issue-1195845543.

**Comment by Rebits** [comment]
### 03/05/2022

Regarding message:
```
2022/04/07 08:14:25 wazuh-agent[3860] exec.c:65 at ReadExecConfig(): ERROR: (1313): Invalid active response config: 'shared/ar.conf'.
```

The content of ar.conf is the following:
```

restart-wazuh0 - restart-wazuh - 0
restart-wazuh0 - restart-wazuh.exe - 0
firewall-drop0 - firewall-drop - 0
firewall-drop5 - firewall-drop - 5
```

**Comment by Deblintrake09** [comment]
## Update 2022/07/28

Launching tests from Jenkins Instance, first run fails, but al subsequent launches pass.
[Machine Report first run fails ](https://ci.wazuh.info/view/Tests/job/Test_integration/29886) :green_circle: 
![imagen](https://user-images.githubusercontent.com/14501079/181572304-e6876736-bc10-4ce7-a01a-2d09221d8aa4.png)
![imagen](https://user-images.githubusercontent.com/14501079/181572409-6160ef43-c1de-47d6-ab29-d7325f059f34.png)
![imagen](https://user-images.githubusercontent.com/14501079/181572463-40c0a1fd-875c-4b63-9c14-a410c9370c61.png)

**Comment by Deblintrake09** [comment]
There was a bug on the windows agent when it was stopped. It caused the fails in this tests. It was fixed by Wazuh/Wazuh PR [#14486](https://github.com/wazuh/wazuh/pull/14486) for 4.4.
Waiting for merge to close this.

**Comment by Deblintrake09** [comment]
### Testing done
| Version | Commit | Results|Notes|
|---|---|---|
| 4.4 || :large_blue_circle|
|4.5| | [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30467/)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30468/)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30469/)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30470/)[:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30471/) |
